shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/domain/analytics/stage_implement_unsupported_modal/StageImplementUnsupportedModalHiddenHyperskillAnalyticEvent.kt

```diff
@@ -12,18 +12,16 @@ import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticTar
  * JSON payload:
  * ```
  * {
- *     "route": "/projects/{PROJECT_ID}/stages/{STAGE_ID}/implement",
+ *     "route": "/study-plan",
  *     "action": "hidden",
  *     "part": "stage_implement_unsupported_modal",
  *     "target": "close"
  * }
  * ```
  * @see HyperskillAnalyticEvent
  */
-class StageImplementUnsupportedModalHiddenHyperskillAnalyticEvent(
-    route: HyperskillAnalyticRoute
-) : HyperskillAnalyticEvent(
-    route,
+object StageImplementUnsupportedModalHiddenHyperskillAnalyticEvent : HyperskillAnalyticEvent(
```

Comment:
Hm, interesting how `object` analytic event will be rendered in our docs https://hyperskill.github.io/mobile-app (previously we only used classes)